com.mikrotik.WinBox.appdata.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>com.mikrotik.WinBox</id>
+    <launchable type="desktop-id">com.mikrotik.WinBox.desktop</launchable>
+    <name>WinBox</name>
+    <developer_name>SIA Mikrotīkls</developer_name>
+    <summary>RouterOS administration tool</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LicenseRef-proprietary</project_license>
+    <description>
+        <p>
+            WinBox is a small utility that allows the administration of
+            MikroTik RouterOS using a fast and simple GUI. All WinBox
+            interface functions are as close as possible mirroring the
+            console functions, that is why there are no WinBox sections
+            in the manual. 
+        </p>
+        <p>
+            NOTE: This wrapper is not verified by, affiliated with, or
+            supported by MikroTik.
+        </p>
+    </description>
+    <url type="homepage">https://mikrotik.com</url>
+    <url type="help">https://help.mikrotik.com/docs/display/ROS/WinBox</url>
+    <url type="bugtracker">https://mikrotik.com/support</url>
+    <screenshots>
+        <screenshot type="default" environment="gnome">
+            <image>winbox-main-light@2X.png</image>
```

Comment:
these need to be image URLs

Reply:
Use the links from the PR now https://raw.githubusercontent.com/flathub/flathub/5f148399d3bb158c5afbd63f107aab741850d0d4/winbox-list-dark%402X.png then update it later once repo is created